src/routes.json

```diff
@@ -199,7 +199,7 @@
     },
     {
         "name": "preview",
-        "pattern": "^/preview(/editor|(/\\d+(/editor|/fullscreen)?)?)?/?$",
+        "pattern": "^/preview(/editor|(/\\d+(/editor|/fullscreen)?)?)?/?(\\?.*)?$",
```

Comment:
are we positive that querystring is part of the route, in both node AND fastly?

Reply:
Chris pointed me to https://docs.fastly.com/guides/vcl-tutorials/vcl-regular-expression-cheat-sheet which is helpful!